.github/workflows/lint.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Install Dependencies
         run: pip install -r requirements/lint.txt
       - name: Black Format Check
-        run: black --check . --exclude samples/
+        run: black --check .
```

Comment:
are we sure this is what we want? Changing the format of our samples can have some unintended effects on some codemods.

Reply:
This is now excluded in `pyproject.toml` since that affects all environments and not just GitHub actions.